### ISSUE TITLE
508-enhancement [COGNITION]: Direct Deposit Component - Consider adding a figure + figcapture pattern for the check image

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va-forms-system-core#255]
ORIGINAL TICKET: https://github.com/department-of-veterans-affairs/va.gov-team/issues/18439

_**IF WE WORK ON THIS TICKET, WE MAY WANT TO CONSIDER PULLING IN OTHER ACCESSIBILITY TICKETS RELATED TO DIRECT DEPOSIT TO WORK ON AT THE SAME TIME: department-of-veterans-affairs/va-forms-system-core#313, department-of-veterans-affairs/va-forms-system-core#316, AND department-of-veterans-affairs/va-forms-system-core#317.**_

@1Copenut commented on [Tue Jan 12 2021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18439)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor, Crystabel_

## Details

I have a hunch that our alt text description for the check image would be beneficial for all users. We assume a lot that users can see and understand the image explainer for finding routing number and bank account number.

I took some time to experiment with a figure + figcapture pattern as the Thoughtbot team outlined in this article: https://thoughtbot.com/blog/alt-vs-figcaption

It works really well with screen readers and would give all users an image + text explanation of how to find relevant information. Tagging @CrystabelReiter for awareness, because this would be a visual adjustment too.

## Proposed Solution (if known)

```html
<figure>
  <img src="/" alt="Personal checks have all the information to set up direct deposit" />
  <figcaption>
    Your bank’s routing number is listed along the bottom-left edge of a personal check. Your account number is listed to the right of that. Routing numbers must be nine digits and account numbers can be up to 17 digits.
  </figcaption>
</figure>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

---

@CrystabelReiter commented on [Thu Jan 14 2021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18439#issuecomment-760462172)

@1Copenut Can you tell me more about the visual adjustment? How would the image look different from how it does now? Thanks!

---

@1Copenut commented on [Thu Jan 14 2021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18439#issuecomment-760506373)

@CrystabelReiter If we went this direction, the `<figcaption>` text could be presented underneath the image of the check, maybe centered or slightly indented from the left. I've been thinking about this pattern off and on for a while, as a way to keep `alt` text streamlined, but offer all users a more in-depth explanation for images like checks, maps, etc. that might benefit from it.

---

@CrystabelReiter commented on [Thu Jan 14 2021](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18439#issuecomment-760520249)

@1Copenut Thanks for the further explanation. :) ! I'll talk to Chris and Carol and see how we can fit it in to the next sprint

**Comment by obliviga** [comment]
Hi @taylorkaren, here is what the proposed solution will look like:
![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/2a6e52f8-5346-4440-a9c8-0f882dc0d9f7)

The only visual change that is being added is the blue text under the check image. Do you know if there's someone that should approve how this looks? I think Crystabel Reiter was the previous designer on this, but I don't think they are with Ad Hoc anymore.

**Comment by taylorkaren** [comment]
@obliviga: I will ask Matt about this in our PO sync call tomorrow.

**Comment by taylorkaren** [comment]
@obliviga: Guidance from Matt on how to proceed with this ticket:
	•	Does not need to be blue text, it can be black. 
	•	More spacing between the image and the text (4px, half spacing unit)
	•	Ask the content team to review the text (Danielle Thierry or Beth Potts) then we can probably just make the change (Matt to confirm this with Kevin).

**Comment by obliviga** [comment]
Fix has been verified in production: https://adhoc.slack.com/files/U03ETG3PG6T/F03LQ2GFZJ9/direct_deposit_functionality_prod.mov
